Dockerfile

```diff
@@ -86,6 +86,12 @@ ENV PIP_DISABLE_PIP_VERSION_CHECK=true \
     PYTHONIOENCODING=utf8 \
     PYTHONUNBUFFERED=1
 
+
+# Update to the latest packages for the base image
+RUN apk update &&\
```

Comment:
since this is from `python:3.10-alpine`, is this a problem with the Alpine image as well?

Reply:
It is a current problem with the Alpine image for which there aren't new builds yet.
See https://github.com/docker-library/python/issues/761

Reply:
Makes sense. This looks fine then

Reply:
Thanks!
Resolving the conversation then. Feel free to re-open it or open a new one.